package.json

```diff
@@ -24,21 +24,18 @@
     "index.js"
   ],
   "devDependencies": {
-    "babel": "^6.1.18",
-    "babel-preset-es2015": "^6.1.18",
-    "babel-tape-runner": "^2.0.0",
+    "ava": "^0.11.0",
```

Comment:
Each of the 5.x, 4.x, and 0.12.x AppVeyor builds now pass :+1: 

https://ci.appveyor.com/project/stylelint/stylelint-config-wordpress/build/98